cpp/include/cudf/table/primitive_row_operators.cuh

```diff
@@ -143,6 +143,19 @@ class row_equality_comparator {
                                                           rhs_row_index);
   }
 
+  /**
+   * @brief Compares the specified rows for equality.
+   *
+   * @param lhs_index The index of the first row to compare (in the lhs table)
+   * @param rhs_index The index of the second row to compare (in the rhs table)
+   * @return true if both rows are equal, otherwise false
+   */
+  __device__ bool operator()(cudf::experimental::row::lhs_index_type lhs_index,
```

Comment:
```suggestion
  __device__ bool constexpr operator()(cudf::experimental::row::lhs_index_type lhs_index,
```

Reply:
My bad, it's not needed. I realized this function just calls the other operator. All good. Please proceed.